airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/jobs/registry.py

```diff
@@ -52,6 +52,35 @@ def remove_stale_metadata_partitions():
     remove_stale_metadata_partitions_op()
 
 
+@op(required_resource_keys={"latest_metadata_file_blobs"})
+def remove_latest_metadata_partitions_op(context):
+    """
+    This op is responsible for removing for latest metadata files. (Generally used to reprocess metadata files).
```

Comment:
```suggestion
    This op is responsible for removing for latest metadata partitions. (Generally used to reprocess metadata files).
```
The difference between these 2 is what made it so hard for me to comprehend for so long, so keeping it correct :D